LayoutTests/imported/w3c/web-platform-tests/trusted-types/HTMLScriptElement-internal-slot.html

```diff
@@ -7,24 +7,17 @@
 </head>
 <body>
 <script>
+  // This policy allows document.write to work as that's not relevant to what's actually being tested here.
+  trustedTypes.createPolicy('default', {createHTML: s => s});
```

Comment:
Exported at https://github.com/web-platform-tests/wpt/pull/46308

Change is needed as the existing method was flaky and didn't reproduce properly in WebKit and Gecko.